### DISCUSSION TITLE
default comment module is not loaded.

### DISCUSSION THREAD

**Discussion by mncmncme** [requarks/wiki#5200] [Error / Bug Report] [answered]
## Environment:
 - Docker : requarks/wiki:latest (version: 2.5.278)
 - Database : both MariaDB and PostgreSQL
 - Immediately after installation
 
## Error Desc:
The default comments at the bottom of the page are not loaded. However, the external comment module (commneto, disqus) is loaded.
### Browser screen
> ![image](https://user-images.githubusercontent.com/65083368/164133499-d7729b1a-1606-4891-9e22-d01f09f7bcdc.png)

### Developer panel in browser
There is nothing in the _div_ tag
> ![image](https://user-images.githubusercontent.com/65083368/164133786-c0bdbc14-bcff-46b7-bded-0109a48bd94d.png)

### Compare to version 2.4.107
Loaded normally
> ![image](https://user-images.githubusercontent.com/65083368/164133273-3ab98cf3-daee-44ca-a5e9-906ae48bcae4.png)

**Comment by mncmncme** [comment]
I tested some of the past versions. It's all working. It seems to be a recent problem.

- [x] 2.5.197
- [x] 2.5.201
- [x] 2.5.256
- [x] 2.5.272
- [x] 2.5.277

**Comment by hiverlab-george** [comment]
Facing the same issue.

The following error shows up in console when 'New comment' is clicked.
![image](https://user-images.githubusercontent.com/81133380/164304235-e1ba0c16-2b65-4f4f-ad3d-8b2be26550bb.png)

**Comment by akoscomp** [comment]
I have the same problem. When I upgraded from 2.5.277 to 2.5.278 the comment feature became inactive.

When I restored this commit, the comment feature works again:
https://github.com/requarks/wiki/commit/a647626a51da8aec5adf35123af5f1dce656fa48

**Reply by TaiiHu** [reply]
work for me, thanks

**Comment by EyalSeagull** [comment]
@NGPixel , I'm having the same problem. After installing version 2.5.278 the built in comment engine and the Disqus comment engine are showing an empty comment space with no option to add comments.

![image](https://user-images.githubusercontent.com/19786419/164896914-35dbcecc-a10d-4ffa-b2cf-488cb93b7512.png)

**Reply by bpwhite** [reply]
Same

**Comment by Genuifx** [comment]
I get the same error.

wikijs version 2.5.278
![image](https://user-images.githubusercontent.com/10156994/165221192-1b4e4c5c-269d-47eb-8473-b6a718e89c14.png)

**Comment by EyalSeagull** [comment]
Hi, try changing the following line in the **/wiki/server/controllers/common.js** file:

Current:
`const shouldDisplayComments = WIKI.config.features.featurePageComments && WIKI.data.commentProvider.codeTemplate`

New:
`const shouldDisplayComments = WIKI.config.features.featurePageComments`

**Reply by tejo-ak** [reply]
I confirm this approach works. 
Changing && to II also work
`const shouldDisplayComments = WIKI.config.features.featurePageComments || WIKI.data.commentProvider.codeTemplate`

I wonder if it is the solution, or is it a work around?

**Reply by EyalSeagull** [reply]
Hi @tejo-ak , using `||` means that even if `WIKI.config.features.featurePageComments` is `False` the comments can still be shown (in case `WIKI.data.commentProvider.codeTemplate` is `True`). Is it a desirable funcionallity?

**Comment by NGPixel** [comment, marked as answer]
Fixed in 2.5.279
